docs/modules/ROOT/pages/auth/authorization/where.adoc

```diff
@@ -46,6 +46,8 @@ Where is used on the following operations;
 - `DELETE`
 
 
+Note that `where` rules are not supported when applied over nested xref::type-definitions/unions.adoc[unions] or xref::type-definitions/interfaces.adoc[interfaces].
```

Comment:
**suggestion (blocking):** this will also apply to `allow` and `bind` as well, as far as I'm aware, so this should also be documented, either on those pages as well, or at the top-level.

**suggestion (non-blocking):** maybe use an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/) for this comment wherever it ends up?

Reply:
WE HAVE ADMONITION??
Ooh yiss